client/src/components/TodoCardList/Item/index.js

```diff
@@ -1,17 +1,29 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Card, CardBody, Input } from 'mdbreact'
+import { Card, CardBody } from 'mdbreact'
+
+import styles from 'src/components/TodoCardList/Item/styles.css'
```

Comment:
@DDKatch can't we do 
`import styles from 'styles.css'`
or
`import styles from './styles.css'`
?